docs/apis/common/index.md

```diff
@@ -1350,3 +1350,397 @@ You may specify program arguments before the job is executed. The plan visualiza
 the execution plan before executing the Flink job.
 
 {% top %}
+
+Metrics
```

Comment:
It's also quite big and blows up the Basic API page. It takes up a quarter of the page alone.

I'll move it.